### DISCUSSION TITLE
ISR (Incremental Static Regeneration) with Cloudflare Pages

### DISCUSSION THREAD

**Discussion by rayblair06** [nuxt/nuxt#27867] [General]
Has anyone investigated or found possible solutions for achieving ISR (Incremental Static Regeneration) with Cloudflare Pages?

We are exploring options to scale our SSG application because our estimated build times are quite high. We will have over 6000 pages, which may take around 30 minutes per build, and changes to our infrastructure might not be possible.

**Comment by TheAlexLichter** [comment]
This should probably be discussed in in https://github.com/unjs/nitro. Please raise it there 😋

**Reply by rayblair06** [reply]
Hey, @manniL. My mistake, I'll close this and take my enquiry over there :)

https://github.com/unjs/nitro/discussions/2568 for future followers.
